reactioncommerce/reaction
# Fixed Inconsistent Label spacing on PDP

Resolves #3042 

 I have added a custom style class in `products.less` to fix the label badges anomaly.

#### How to test
- Login in as admin
- Add a new product 
- Enable Inventory tracking
- Set `Warn at`  value to be more than the `Quantity` value to trigger getting the `Backorder` label badge
- Archive the variant to also trigger the `Archived` label badge.
- Observe that the labels are currently displayed with the desired spaces.
